pymeasure/display/placeholders.py

```diff
@@ -0,0 +1,63 @@
+import string
+import re
+
+from datetime import datetime
+
+builtin_placeholders = {'time' : lambda: datetime.now().strftime('%Hh%Mm%Ss'),
+                        'date' : lambda: datetime.now().strftime('%y.%m.%d'),
+                        }
+
+class PlaceholderReplacer():
+    """ Allows to replace sequence of characters in a string with information
+    or parameter values.
+
+    :param inputs: Reference to the input widget
+    """
+
+    def __init__(self, inputs):
+        
+        self.pattern = r'\<.*?\>'
```

Comment:
Would it be an idea to use the built-in `format` method of python strings. Might give some additional flexibility.